spec/controllers/ctc/ctc_pages_controller_spec.rb

```diff
@@ -96,7 +96,7 @@
   describe "#puerto_rico" do
     context "when the puerto rico features has launched" do
       before do
-        allow(Flipper).to receive(:enabled?).with(:puerto_rico_home_location).and_return true
+        Flipper.enable :puerto_rico_home_location
```

Comment:
is there an explicit reason or thought process on enabling directly vs stubbing the enabling? Does Flipper docs recommend one way over the other?

Reply:
`allow .. to receive .. and_return` will turn on/off *every* feature flag currently in effect, which can cause bad results

`allow .. to receive .. with .. and_return` doesn't scale well with multiple features to flip, because it will complain if `enabled?` is called with different arguments in a different order -- "expected to be called with puerto_rico but was called with eitc"

so we could beef up the stubbing even further to compensate for that, but all we're really looking for is an in-memory list of features that are flipped on that gets reset every test, and Flipper provides that with `Flipper.instance = Flipper.new(Flipper::Adapters::Memory.new)`